src/core/createCookie.js

```diff
@@ -10,13 +10,24 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+import { isNonEmptyString } from "../utils";
+
 export default (cookieProxy, componentNamespace) => {
+  const validateNamespace = () => {
+    if (!isNonEmptyString(componentNamespace)) {
+      throw Error(
+        "No cookie namespace.  Please define 'abbreviation' on the component."
+      );
+    }
+  };
+
   return {
     /**
      * Returns a value from the Alloy cookie for a given key.
      * @param {string} key
      */
     get(key) {
+      validateNamespace();
```

Comment:
Is there a reason why we don't just validate it once before returning the cookie object instead of on every `get`, `set`, and `remove`?

Reply:
I didn't want to throw an error for a component that doesn't use the cookie.

Reply:
In my opinion, it isn't worth it. If `componentNamespace` isn't defined, I'd like to hear about it ASAP regardless of whether the cookie is used. In any case, if you choose to keep it, maybe add a comment for future devs because this is something I would likely change if I didn't understand your intent.

Reply:
I don't have a strong opinion either way, so I have made your suggested change.